apis/slo/v1alpha1/pod.go

```diff
@@ -14,33 +14,32 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-package extension
+package v1alpha1
```

Comment:
I have a little doubt: will some codes in the extension package be adjusted according to this new layout? For example, slo-related extension protocols are putting into slo/v1alpha1, and scheduling-related extension protocols are putting into scheduling/v1alpha1.

Reply:
i think it depends, for example the `CPUBurstConfig` struct, it is slo-relate and specific for slo config, which already defined in slo/v1alpha1 and no one else will use it except slo-controller/koordlet. It's better to mv related functions to slo/v1alpha1. While for other common slo-related functions like `GetPodQoSClassByName`, which can be used for other modules, maybe should be kept inside extension.

Reply:
If it is determined to adjust the code layout as a whole, I think it is OK.